integration/spark-common-test/src/test/scala/org/apache/carbondata/integration/spark/testsuite/complexType/TestCompactionComplexType.scala

```diff
@@ -1068,4 +1068,40 @@ class TestCompactionComplexType extends QueryTest with BeforeAndAfterAll {
     sql("Drop table if exists adaptive")
   }
 
+  test("Test major compaction for struct of array type") {
+    sql("DROP TABLE IF EXISTS carbon")
```

Comment:
1. Include dropping of table in afterAll also
2. Give the test case description as below
`Test major compaction with dictionary include for struct of array type`